tensorflow-core/tensorflow-core-api/src/test/resources/saved_model_using_python/readme.txt

```diff
@@ -0,0 +1,45 @@
+Saved model created using Python @tf.function, tensorflow version 2.3.0.
+
+Python code used to create saved model below:
```

Comment:
Should we just check this in as a python file?

Reply:
@Shajan  , what do you think? I think it could be a Python file, as long as we add a notice on top of that the file is only there for reference and is not being used anywhere in the code base.

Reply:
I've moved forward and changed the file type to Python, please let me know if you think of any reason I should revert it to a text file.